Documentation/contributing/development/dev_setup.rst

```diff
@@ -579,12 +579,34 @@ Debugging
 Datapath code
 ^^^^^^^^^^^^^
 
-The tool ``cilium monitor`` can also be used to retrieve debugging information
-from the eBPF based datapath. Debugging messages are sent if either the
-``cilium-agent`` itself or the respective endpoint is in debug mode. The debug
-mode of the agent can be enabled by starting ``cilium-agent`` with the option
-``--debug`` enabled or by running ``cilium config debug=true`` for an already
-running agent. Debugging of an individual endpoint can be enabled by running
+The image below shows the options that could be used as startup options by
+``cilium-agent`` (see upper blue box) or could be changed at runtime by running
+``cilium config <option(s)>`` for an already running agent (see lower blue box).
+For instance, if you want to enable the debug mode (including the load
+balancer), you may start ``cilium-agent`` with ``--debug-verbose=datapath`` or
+run ``cilium config debug=true debugLB=true`` for an already running agent. For
+each option, there is a helper function which can be used by the developer to
+print debugging information in the tool ``cilium monitor``: ``cilium_dbg()`` for
+``DEBUG`` and ``cilium_dbg_lb()`` for ``DEBUG_LB``. 
+
+.. image:: _static/cilium-debug-datapath-options.svg 
+  :align: center
+  :alt: Cilium debug datapath options
+
+.. note::
+
+   You can also use ``printk()`` to debug the datapath outside the ``cilium
+   monitor``.  In this case, ``bpftool prog tracelog`` can be used to retrieve
+   debugging information from the eBPF based datapath.  However, it is recommended
+   to use ``cilium_dbg()``. Both ``cilium_dbg()`` and ``printk()`` helpers are
```

Comment:
We would likely recommend `cilium_dbg()` over `prink()` because it is better integrated with the rest of the debug messages available from `cilium monitor`. But I don't think we should discourage people from using `prink()` if they want: When it comes to debugging, you never have too many tools. I think the `outside [of] the cilium monitor` sums up the idea already, so I'd just remove that sentence about the recommendation.

```suggestion
   debugging information from the eBPF based datapath.
   Both ``cilium_dbg()`` and ``printk()`` helpers are
```